### DISCUSSION TITLE
Comments in API Filter rules

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#2831] [Q&A]
@ganigeorgiev 

Is it feasible to add the ability to comment into the api filter rules?  Perhaps just `/* */` notation.

**Comment by dschissler** [comment]
I was also thinking that perhaps upon save that you could preparse the view SQL and API Rules text to improve runtime performance just a bit.

**Comment by ganigeorgiev** [comment]
API rules comments is in the roadmap and will be implemented in the near future.
